### PR TITLE
Allow using psr/log v3

### PR DESCRIPTION
**Pull request by Qrzysio**
Why not using the third version of popular `psr/log` dependency?

`"psr/log": "^1.0 || ^2.0 || ^3.0",`

### CODE REVIEW

**Comment by harikt** [comment]
Hi there, 

I have not been taking very close look into the psr changes. ie why.

Thank you for the PR and it looks good to me.

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     ],
     "require": {
         "php": ">=8.1",
-        "psr/log": "^1.0 || ^2.0",
+        "psr/log": "^1.0 || ^2.0|| ^3.0",
```
Comment (harikt) [inline composer.json]: One thing I wished is if there is a space between `2.0` and `|` . Not a big deal just pointing.

Comment (koriym) [inline composer.json]:
```suggestion
        "psr/log": "^1.0 || ^2.0　|| ^3.0",
```

**Comment by harikt** [comment]
Thank you.

**Comment by harikt** [comment]
Interesting the test failed after merge.

File: composer.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
     ],
     "require": {
         "php": ">=8.1",
-        "psr/log": "^1.0 || ^2.0",
+        "psr/log": "^1.0 || ^2.0　|| ^3.0",
```
Comment (kenjis) [inline composer.json:26]: Should be `"^1.0 || ^2.0 || ^3.0"`.
There is a zero width space (`　`) after `2.0`.

**Pull request merged**